test/functional/test-ads-config.js

```diff
@@ -28,7 +28,7 @@ describe('test-ads-config', () => {
     delete adConfig.fakead3p;
     const keys = Object.keys(adConfig);
     for (let i = 0; i < keys.length - 1; i++) {
-      expect(keys[i]).to.be.below(keys[i + 1]);
+      expect(keys[i] < keys[i + 1]);
```

Comment:
Why doesn't `.below` work? http://chaijs.com/api/bdd/#method_below

Reply:
From the link, `.below` works only on the `Number` type. Here, `keys[i]` is a `string`.